src/main/java/com/minecolonies/commands/SplitCommand.java

```diff
@@ -0,0 +1,108 @@
+package com.minecolonies.commands;
+
+import net.minecraft.command.CommandException;
+import net.minecraft.command.ICommandSender;
+import net.minecraft.server.MinecraftServer;
+import net.minecraft.util.math.BlockPos;
+import org.jetbrains.annotations.NotNull;
+
+import javax.annotation.Nullable;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * A command that has children.
+ */
+public abstract class SplitCommand implements ISubCommand
+{
+
+    private final String[] parents;
+
+    public SplitCommand(@NotNull String[] parents)
+    {
+        this.parents = parents;
+    }
+
+    @NotNull
+    @Override
+    public String getCommandUsage(@NotNull final ICommandSender sender)
+    {
+        final Map<String, ISubCommand> childs = getSubCommands();
+        final StringBuilder sb = new StringBuilder().append('/');
+        for (String parent : parents)
+        {
+            sb.append(parent).append(' ');
+        }
+        sb.append('<');
+        boolean first = true;
+        for (String child : childs.keySet())
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'child' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)